settings.gradle

```diff
@@ -1,7 +1,7 @@
+include ':counter'
```

Comment:
Since I saw categorization in other `gradle` files, I will recommend categorizing samples vs library modules here.

Reply:
@tasomaniac [Great idea!](https://jeroenmols.com/blog/2019/04/02/modularizationexample/) 🎉 

```Groovy
{
    include ':libraries:mvrx'
    include ':libraries:testing'

    include ':samples:counter'
    include ':samples:dogs'
    include ':samples:hellodagger'
    include ':samples:sample'
    include ':samples:todomvrx'
}
```

But I rather do this in a sole PR only for it because lots of files would be just renamed and make a lot of noise at the review. Also it would require [`git log --follow`](https://stackoverflow.com/questions/4393527/why-might-git-log-not-show-history-for-a-moved-file-and-what-can-i-do-about-it) to track code changes.

Does it make sense @elihart @tasomaniac?

Reply:
What I meant was to simply put header comments like `// Samples` and separate them. 😊

Reply:
@tasomaniac agreed, resolved at #364, and rebased #367 over it.